frontend/src/components/activity.js

```diff
@@ -3,17 +3,55 @@ import Col from "react-bootstrap/esm/Col";
 import Row from "react-bootstrap/esm/Row";
 import Image from "react-bootstrap/Image";
 import avatarPlaceholder from "../asset/avatar_placeholder.png";
+import { domain } from "../services/apis";
+
+const Activity = ({ activity }) => {
+  const { content_type, content_object } = activity;
+  const FOLLOW = 9;
+  const LESSON = 12;
+  const date = new Date(activity.activity_date);
+  let avatar, message, hours, days;
+
+  const seconds = (Date.now() - date.getTime()) / 1000;
+  let minutes = Math.round(seconds / 60);
+  let timeMessage = `${minutes} minutes ago`;
+
+  if (minutes >= 60) {
+    hours = Math.floor(minutes / 60);
+    minutes = minutes % 60;
+    timeMessage = `${hours} hours ${minutes} minutes ago`;
+  }
+
+  if (hours >= 24) {
+    days = Math.floor(hours / 24);
+    hours = hours % 24;
+    timeMessage = `${hours} hours ${minutes} minutes ago`;
+  }
```

Comment:
Dont re invent the wheel. If you think this is already implemented somewhere, there is a big possibility that it has a package already so look for it.
https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/02-fromnow/